Functions/QualityControl/xASL_qc_CleanOldQC.m

```diff
@@ -0,0 +1,89 @@
+function [x] = xASL_qc_CleanOldQC(x, ScanType, bRemoveCurrentSession)
+%xASL_qc_RemoveOutdatedQC Removes outdated QC parameters, loads QC from JSON and cleans the current session if necessary
+%
+% FORMAT: [x] = xASL_qc_RemoveOutdatedQC(x, ScanType[, bRemoveCurrentSession)
+%
+% INPUT:
+%   x 	                  - structure containing fields with all information required to run this submodule (REQUIRED)
+%   ScanType              - Type of the data - can be 'ASL' or 'func' (REQUIRED)
+%   bRemoveCurrentSession - Removes the current session completely (OPTIONAL, DEFAULT = false)
+%
+% OUTPUT:
+%   x        - same as input
+% -----------------------------------------------------------------------------------------------------------------------------------------------------
+% DESCRIPTION: This function removes the old and outdated QC parameters. Specifically parameters for ASL and func which are now given in a subfield by session.
+%              It also loads further sessions from the disc and if necessary, it cleans the current session completely.
+%
+% EXAMPLE: x = xASL_qc_RemoveOutdatedQC(x, 'ASL');
+% __________________________________
+% Copyright (C) 2015-2023 ExploreASL	
+
+if nargin < 2
+	error('Two parameters are requied');
```

Comment:
minor spelling error

Reply:
Good catch. Fixed and added to the next commit in the queue.